src/ts/component/popup/shortcut.tsx

```diff
@@ -260,8 +260,6 @@ class PopupShortcut extends React.Component<I.Popup, State> {
 							{ com: '->',				 name: UtilCommon.sprintf(translate('popupShortcutMarkdownWhileTypingInserts'), '→') },
 							{ com: '<-',				 name: UtilCommon.sprintf(translate('popupShortcutMarkdownWhileTypingInserts'), '←') },
 							{ com: '--',				 name: UtilCommon.sprintf(translate('popupShortcutMarkdownWhileTypingInserts'), '—') },
-							{ com: '—>',				 name: UtilCommon.sprintf(translate('popupShortcutMarkdownWhileTypingInserts'), '⟶') },
```

Comment:
Very similar to line 257 and 258

Reply:
It's not duplicate, it's mdash and ndash.